lib/server.go

```diff
@@ -329,12 +332,12 @@ func NewServer(
 
 	// Create a BitcoinManager so that transactions that exchange BitClout for Bitcoin
 	// can be properly validated.
-	_bitcoinManager, err := NewBitcoinManager(
-		_db, _params, timesource, _bitcoinDataDir, _incomingMessages,
-		_bitcoinConnectPeer)
-	if err != nil {
-		return nil, errors.Wrapf(err, "NewServer: Problem initializing BitcoinManager")
-	}
+	//_bitcoinManager, err := NewBitcoinManager(
+	//	_db, _params, timesource, _bitcoinDataDir, _incomingMessages,
+	//	_bitcoinConnectPeer)
+	//if err != nil {
+	//	return nil, errors.Wrapf(err, "NewServer: Problem initializing BitcoinManager")
+	//}
```

Comment:
As per my earlier comment, not sure why we need to kill the bitcoin manager in order to make postgres work.